lib/active_record/session_store.rb

```diff
@@ -33,6 +37,60 @@ def create_table!
         end
         connection.add_index table_name, session_id_column, :unique => true
       end
+
+      def determine_serializer
```

Comment:
Maybe rename this to `serializer_class` so that when you call a method on it it's like calling method on an object/class and not on an action?